blocks/footer/footer.js

```diff
@@ -22,6 +22,45 @@ export default async function decorate(block) {
     }
   });
 
-  await decorateIcons(footer);
+  const firstFooterDiv = footer.querySelector('.footer-links');
+  firstFooterDiv.classList.add('footer-menu');
+  const headings = firstFooterDiv.querySelectorAll('ul > li > strong');
+  headings.forEach((div) => {
+    div.classList.add('heading');
+  });
+
+  const menu = firstFooterDiv.querySelectorAll('ul > li > ul');
+  menu.forEach((div) => {
+    div.classList.add('menu');
+  });
+
+  const menuHeadings = footer.querySelectorAll('.heading');
+  const menuLists = footer.querySelectorAll('.menu');
+  menuHeadings.forEach((heading, index) => {
+    const caret = document.createElement('span');
+    caret.classList.add('caret');
+    caret.innerHTML = '&#8964;';
```

Comment:
*nitpicking* This could have been added to the CSS via a `::before` or equivalent to decouple behavior from visual rendering